scaleway/resource_baremetal_server.go

```diff
@@ -219,7 +220,7 @@ If this behaviour is wanted, please set 'reinstall_on_ssh_key_changes' argument
 				},
 			},
 		},
-		CustomizeDiff: func(_ context.Context, diff *schema.ResourceDiff, i interface{}) error {
+		CustomizeDiff: func(ctx context.Context, diff *schema.ResourceDiff, i interface{}) error {
```

Comment:
Could you extract this function in a named function? And combine it with a sequence ?